docker/community/docker-compose.yml

```
@@ -138,27 +139,24 @@ services:
     ports:
       - ${WS_PORT}:${WS_PORT}
 
-  web:
-    image: 'ghcr.io/novuhq/novu/web:2.1.1'
+  dashboard:
+    image: 'ghcr.io/novuhq/novu/dashboard:2.1.1'
     depends_on:
       - api
       - worker
-    container_name: web
+    container_name: dashboard
     restart: unless-stopped
     logging:
       driver: 'json-file'
       options:
         max-size: '50m'
         max-file: '5'
     environment:
-      REACT_APP_API_URL: ${API_ROOT_URL}
-      REACT_APP_ENVIRONMENT: ${NODE_ENV}
-      REACT_APP_WIDGET_EMBED_PATH: ${WIDGET_EMBED_PATH}
-      REACT_APP_IS_SELF_HOSTED: 'true'
-      REACT_APP_WS_URL: ${REACT_APP_WS_URL}
+      VITE_API_HOSTNAME: ${VITE_API_HOSTNAME}
+      VITE_SELF_HOSTED: ${VITE_SELF_HOSTED}
```

Should we keep it as true as we had before.
it could not be false

```suggestion
      VITE_SELF_HOSTED: true
```